internal/version/version_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_GetVersion(t *testing.T) {
```

Comment:
Maybe this test function is not needed, since it doesn't increase coverage, see code coverage report of [last run](https://app.codecov.io/gh/oras-project/oras/pull/977/indirect-changes?src=pr&el=tree-more&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=oras-project), line 34 is already covered.